catalog/content/docs/src/main/resources/_contents/integrating-contents.adoc

```diff
@@ -64,7 +64,7 @@ Compatibility is reflected in version numbers.
 
 ==== Design
 
-The {branding} Content Framework design consists of several major components, namely: endpoints, input transformers, the core Content Framework, storage providers, storage plugins, and Content plugins.
+The ${ddf-content} Framework design consists of several major components, namely: endpoints, input transformers, the core Content Framework, storage providers, storage plugins, and Content plugins.
 
 The endpoints provide external clients access to the Content Framework.
```

Comment:
Replace `access to the Content Framework` with `access to the Framework` or `access to the ${ddf-content} Framework`.